### DISCUSSION TITLE
osu!lazer Editor

### DISCUSSION THREAD

**Discussion by NympheGitHub** [ppy/osu#18255] [Ideas] [answered]
I was trying osu!lazer editor and realized i can not look at maps when i click the editor button on main page. It shows me to create a beatmap instantly which is really annoying for the mappers to be honest. I think lazer's editor should be like stable editor. You should able to look other maps when you are on editor panel. On lazer, you have to open solo panel and then click the map for open to editor. It will get annoying to mappers who use stable right now and when i asked them they mostly agreed with me. 

You can see my main issue on this [video](https://kiyoshi.s-ul.eu/27m2fyzd). (Sorry first seconds are black screen because obs thingy)

**Comment by Axilotl17** [comment, marked as answer]
you have to go into the map selection (in solo), right click on a map (difficulty) and select "edit". it's just a different process than stable.

**Reply by NympheGitHub** [reply]
Yes, thats what i meant by " On lazer, you have to open solo panel and then click the map for open to editor. It will get annoying to mappers " but heard peppy is going to add it eventually so it is okay now.

**Comment by peppy** [comment]
The editor will eventually have a select map screen when you click it from the main menu, for now please follow the instructions above.
